src/styles/utils/function.scss

```diff
@@ -0,0 +1,3 @@
+@function minute-rotate($second-time) {
```

Comment:
why do you need a word rotate? Use more meaningful names. 
Function has to be started with a verb and describe what is going on inside
Ex: calculate-minutes